xtask/src/cov.rs

```diff
@@ -22,7 +22,7 @@ pub fn update(sh: &Shell) -> anyhow::Result<()> {
     let initial_branch = cmd!(sh, "git rev-parse --abbrev-ref HEAD").read()?;
 
     println!("Switch branch");
-    let _ = cmd!(sh, "git branch -D cov-data").run();
+    cmd!(sh, "git branch -D cov-data").run()?;
```

Comment:
This is not the right behavior. Ignoring the error is the intended behavior. On first run, when the branch does not exist yet, this will return an error, preventing us from proceeding with the rest of the task. We also really don’t care about whether this fails or not. Alternatively, we could check for the existence of the branch before running this, but it’s probably not very interesting.